saleor/graphql/core/scalars.py

```diff
@@ -88,3 +88,23 @@ def parse_literal_object(node):
             if field.name.value == "unit":
                 unit = field.value.value
         return Weight(**{unit: value})
+
+
+class UUID(graphene.UUID):
+    @staticmethod
+    def serialize(uuid):
```

Comment:
All of these methods can raise `ValueError` and `TypeError`. Why do you catch only `ValueError`?

Reply:
The issue was to throw ValidationError instead of ValueError. Since in other Scalars we throw GraphQLError I did that here too. Other changes were not requested and I can add them.

Reply:
Do we know when a TypeError is raised? We know about ValueError and this is now handled.